Makefile

```diff
@@ -479,3 +478,8 @@ clean: clean-build
 	@rm -rf ./src/ctl/ui/build
 
 .DEFAULT_GOAL := all
+
+lint: install-tools linter
+	@echo "--- :golang: Running linter on 'src'"
+	./scripts/run-ci-lint.sh $(tools_bin_path)/golangci-lint ./src/...
+	./bin/linter ./src/...
```

Comment:
Do we need a helper script for this? Can we just use golangci-lint config directly?

Also, can't recall - what is `bin/linter` and why do we need it on top of golangci-lint?

Reply:
Not sure but it seems it evaluates some custom rule https://github.com/m3db/m3/blob/master/src/cmd/tools/linter/gorules/rules.go

Reply:
+1 to removing the script wrapper, seems redundant

Reply:
you mean this script https://github.com/m3db/m3/blob/master/scripts/run-ci-lint.sh ? can we?

it adds tag if ENV variable is present
```
TAGS=()
if [[ -n "$GO_BUILD_TAGS" ]]; then
  TAGS=("--build-tags" "${GO_BUILD_TAGS}")
fi

```

maybe I would do it as separate PR in that case. In case we break something and need to rollback.

Reply:
I think it makes sense to hardcode `integration` tag at least, so integration tests would be linted.
It does not look like this is in use right now, but should be cleaned up in a followup.